libraries/api-extractor-model/src/model/DeserializerContext.ts

```diff
@@ -53,13 +53,21 @@ export enum ApiJsonSchemaVersion {
    */
   V_1006 = 1006,
 
+  /**
+   * Add an `initializerTokenRange` field to `ApiProperty` and `ApiVariable` to track the item's
+   * initializer.
+   *
+   * When loading older JSON files, this range is empty.
+   */
+  V_1007 = 1007,
+
   /**
    * The current latest .api.json schema version.
    *
    * IMPORTANT: When incrementing this number, consider whether `OLDEST_SUPPORTED` or `OLDEST_FORWARDS_COMPATIBLE`
    * should be updated.
    */
-  LATEST = V_1006,
+  LATEST = V_1007,
```

Comment:
We needed to introduce `V_1007` in https://github.com/microsoft/rushstack/pull/3435 , so let's bump this one to `V_1008`, since I think these two features should get separate SemVer minor bumps.

Reply:
Sounds good, I'll do this tonight. Thoughts on `ApiInitializerMixin` or PR good as-is?

Reply:
Bumped to `V_1008`!